exercises/collatz-conjecture/test/test_collatz_conjecture.c

```diff
@@ -0,0 +1,51 @@
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Do you need all of these include files?

Reply:
As far as I can tell, none are necessary

Reply:
Nope, they're not necessary.  Copied from another exercise.